frb_example/pure_dart_multi/README.md

```diff
@@ -0,0 +1,8 @@
+## Pure-Dart example for `flutter_rust_bridge (multi blocks of rust Api)`
```

Comment:
```suggestion
## Pure-Dart example for `flutter_rust_bridge` with multi blocks of rust Api
```

Reply:
Fixed in 3917234ba483bcd16ba702640a8d4b806d27d912.